irohad/multi_sig_transactions/state/mst_state.hpp

```diff
@@ -80,7 +80,7 @@ namespace iroha {
     bool isExpired(const DataType &tx,
                    const TimeType &current_time) const override;
 
-   private:
+   protected:
```

Comment:
This looks strange, because this needed only for test purposes(mst_test_helpers.hpp ). May be we can add getter?

Reply:
seems reasonable, although the getter will be also for testing only